packages/teleterm/README.md

```diff
@@ -48,28 +48,34 @@ Prepare Webapps repo
 $ git clone https://github.com/gravitational/webapps.git
 $ cd webapps
 $ yarn install
-$ yarn build-term
-$ yarn package-term
+$ yarn build-and-package-term
 ```
 
 The installable file can be found in `/webapps/packages/teleterm/build/release/`
 
 ## Development
 
-**Make sure to run `yarn build-term` first** (as described above) before attempting to launch the
-app in the development mode. That's because Electron is running its own version of Node. That
-command will fetch native packages that were built for that specific version of Node.
+**Make sure to run `yarn build-native-deps-for-term` first** before attempting to launch the app in
+development mode. That's because Electron is running its own version of Node. That command will
+fetch or build native packages that were made for that specific version of Node.
 
-To launch `teleterm` in the development mode:
+```
+$ cd webapps
```

Comment:
what context are we expecting the users to be in when running this command?  If they are in the directory of this README, wouldn't we want them do cd ../../webapps. I don't like that either but, I'm just curious where they would be "starting" from

Reply:
I think we expect them to be in the root of webapps, I mean at least that's where I run my commands from. `cd webapps` is stated explicitly to follow what the other snippets do. I suspect they mostly do it because some steps require you to be in the teleport repo but others need you to be in webapps.